src/migratePivotFolder.test.ts

```diff
@@ -0,0 +1,38 @@
+import { ContentRecord } from "@activeviam/activeui-sdk";
+import { migratePivotFolder } from "./migratePivotFolder";
+import { migrateUIFolder } from "./migrateUIFolder";
+import { servers } from "./__test_resources__/servers";
+import { smallLegacyPivotFolder } from "./__test_resources__/smallLegacyPivotFolder";
+import { smallLegacyUIFolder } from "./__test_resources__/smallLegacyUIFolder";
+import { uiFolderWithoutCalculatedMeasuresEntry } from "./__test_resources__/uiFolderWithoutCalculatedMeasuresEntry";
+
+describe("migratePivotFolder", () => {
+  beforeAll(() => {
+    const generateId = require("./generateId");
+    let counter = 0;
+    jest.spyOn(generateId, "generateId").mockImplementation(() => {
+      const id = `00${counter}`;
+      counter += 1;
+      return id;
+    });
+  });
```

Comment:
This works but it doesn't feel natural to use `spyOn` when you're essentially just mocking the module.
Can't you use `jest.mock` at the root ?